### DISCUSSION TITLE
Adding ESP32 Arduino Nano support and hitting a "Detect" issue

### DISCUSSION THREAD

**Discussion by nanoflite** [JetBrains/intellij-micropython#287] [General]
Hi,

I'm adding Arduino Nano ESP32 support, but I'm hitting an issue I do not understand.

When clicking 'Detect', `module.microPythonFace` is always null. I'm trying to understand why.

Here's the actual line in the code: https://github.com/JetBrains/intellij-micropython/blob/72432a72df5f26ba7c83facf5adc34e17a115ff8/src/main/kotlin/com/jetbrains/micropython/settings/MicroPythonSettingsPanel.kt#L64

I'm totally new to Kotlin, but I have enough mileage to mostly understand what's going on ;-) I'm looking as to where and how `module.microPythonFace` is set/constructed and why it's null.

Cheers,
Johan

**Comment by elmot** [comment]
There is a big chance that that original board support will gone,
please join the discussion #336
